Makefile

```diff
@@ -84,6 +84,7 @@ TARGETS += util-linux-tools
 TARGETS += v4l-uvc-drivers
 TARGETS += wasmedge
 TARGETS += xe-guest-utilities
+TARGETS += xen-guest-agent
```

Comment:
this needs to be added in `.kres.yaml` and run a `make rekres` to regenerate the Makefile.

Reply:
Ah, didn't read the first line. Fixed it